distribution/std-lib/Standard/src/Image/Data/Image.enso

```diff
@@ -0,0 +1,111 @@
+from Standard.Base import all
+import Standard.Image.Data.Histogram
+import Standard.Image.Data.Image.Internal
+import Standard.Image.Data.Matrix
+
+polyglot java import org.enso.image.data.Image as Java_Image
+
+type Image
+
+    ## The image data type.
+
+       The image is represented with a matrix of rows x columns. Each
+       pixel is represented with a vector of 1 to 4 values (channels).
+       Pixel values are normalized in a range [0.0 .. 1.0].
+    type Image opencv_mat
+
+    ## Return the number of image rows.
+    rows : Integer
+    rows = this.opencv_mat.rows
+
+    ## Return the number of image columns.
+    columns : Integer
+    columns = this.opencv_mat.cols
+
+    ## Return the number of image channels.
+    channels : Integer
+    channels = this.opencv_mat.channels
+
+    ## Get the pixel value indexed by row and column.
+    get : Integer -> Integer -> Vector ! Matrix.Index_Out_Of_Bounds_Error
+    get row column =
+        if (row < 0) || (row >= this.rows) then Error.throw (Matrix.Index_Out_Of_Bounds_Error row) else
+            if (column < 0) || (column >= this.columns) then Error.throw (Matrix.Index_Out_Of_Bounds_Error column) else
+                arr = Java_Image.get this.opencv_mat row column
+                Vector.Vector arr
+
+    ## Calculates the per-element sum of an image and a scalar.
+
+       > Example
```

Comment:
Please fix your formatting of examples like the ones I've done above. Separate examples need to have separate headings.